internal/build/container_ops.go

```diff
@@ -71,24 +69,26 @@ func copyDir(ctx context.Context, ctrClient client.CommonAPIClient, containerID
 	return err
 }
 
-// copyDirWindows provides an alternate, Windows container-specific implementation of copyDir.
+// copyWindows provides an alternate, Windows container-specific implementation of copyDir.
 // This implementation is needed because copying directly to a mounted volume is currently buggy
 // for Windows containers and does not work. Instead, we perform the copy from inside a container
 // using xcopy.
 // See: https://github.com/moby/moby/issues/40771
-func copyDirWindows(ctx context.Context, ctrClient client.CommonAPIClient, containerID string, appReader io.Reader, dst string, stdout, stderr io.Writer) error {
+func copyWindows(ctx context.Context, ctrClient client.CommonAPIClient, containerID string, reader io.Reader, dst string, stdout, stderr io.Writer) error {
```

Comment:
What motivated the rename? Was it just length/clarity?

Reply:
Now this is used to copy both files and dirs

Reply:
In my latest changes, this is now reverted back to `copyDirWindows` as requiring `dst` to be a directory greatly simplifies `xcopy` behavior.